cirq/protocols/measurement_key_protocol.py

```diff
@@ -13,12 +13,20 @@
 # limitations under the License.
 """Protocol for object that have measurement keys."""
 
-from typing import Any
+from typing import Any, Iterable, Union, TYPE_CHECKING, Tuple, Dict, List, \
+    Optional, DefaultDict
 
 from typing_extensions import Protocol
 
 from cirq._doc import document
-from cirq.protocols import has_channel
+from cirq.protocols.channel import has_channel
```

Comment:
I think this is unused (but see other comment).

Reply:
Fixed.